docs/static-bundling.md

```
@@ -0,0 +1,196 @@
+# Run initialization code
+if hasattr(__cribo_module_foo_bar, '__cribo_init'):
+    __cribo_module_foo_bar.__cribo_init()
+```
+
+### Import Rewriting
+
+Import statements for bundled modules are handled specially:
+
+1. **Simple imports** (`import foo`) - Removed entirely as module objects are pre-created
+2. **From imports** (`from foo import bar`) - Converted to attribute access:
+   ```python
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)